src/mixins/createRenderMixin.ts

```diff
@@ -149,6 +151,7 @@ function generateID(cachedRender: RenderMixin<RenderMixinState>): string {
 const widgetClassesMap = new WeakMap<RenderMixin<RenderMixinState>, string[]>();
 
 const createRenderMixin = createStateful
+	.mixin(createI18nMixin)
```

Comment:
That's a tough call. On the one hand, mixing it directly into `createRenderMixin` automatically benefits other out-of-the-box widgets like `createButton`, so that users don't have to store a separate factory somewhere just to use i18n with those widgets. On the other hand, no, not every widget will need to be localized.

At the very least, I'd argue that the out-of-the-box widgets should include `createI18nMixin`. Whether they inherit that through `createRenderMixin` or mix it in themselves depends on whether the additional overhead in `createRenderMixin` does or does not outweigh the inconvenience of users needing to mix `createI18nMixin` explicitly into their own widgets.